src/nnom_layers.c

```diff
@@ -117,7 +117,7 @@ nnom_layer_t *Conv2D(uint32_t filters, nnom_shape_t k, nnom_shape_t s, nnom_padd
 	layer->bias = b;
 	layer->weights = w;
 	layer->output_shift = w->shift;
-	layer->bias_shift = w->shift - b->shift; // bias is quantized to have maximum shift of weights
```

Comment:
The bias_shift is the difference between a product of (input x weight), which is (input_n + weight_n) in Qmn format.
 
I will rather keep it as it is now. 
This is related to a TODO of supporting different Qmn format across layers. 
Please see my [comment](https://github.com/majianjia/nnom/pull/7#issuecomment-468775756) for more explanation.

Reply:
I can somehow understanding your explain, for each of the layer, I know that if its input in format Qim.in, and its output format is Qom.on, and the weights in format Qwm.wn and bias in format Qbm.Qbn, then there is below relation ship:
bias_shift = wn+in-bn
output_shift = wn+in - on
And by the way, I think it's not reasonable to fix each layer's input data to format Q7.0, as my study mnist notebook shows that its not fixed. For the input, it's Q0.7, for th conv1, it's Q2.5,..etc.

If possible, we can arrange a meeting to talk about it.